docs/guides/_toc.json

```
@@ -48,7 +48,7 @@
           "children": [
             {
               "title": "Introduction to Qiskit patterns",
-              "url": "/guides/intro-to-patterns"
+              "url": "/guides/introduction-to-patterns"
```

```suggestion
              "url": "/guides/intro-to-patterns"
```
A change to the file name will break the table of contents link and other referring links - restoring this for now